### PR TITLE
docs: Update list of statistical models for 2022

### PR DESCRIPTION
**Pull request by matthewfeickert**
# Description

Update the list of the published statistical models to include those from 2022 and those from previously published analyses that ATLAS has made public. Following https://pyhf.github.io/public-probability-models

# Checklist Before Requesting Reviewer

- [x] Tests are passing
- [x] "WIP" removed from the title of the pull request
- [x] Selected an Assignee for the PR to be responsible for the log summary

# Before Merging

For the PR Assignees:

- [x] Summarize commit messages into a comprehensive review of the PR

```
* Update the list of published statistical models to include new ones analyses in 2022
  and previous analyses that have had their models made public. Information is gathered
  from https://github.com/pyhf/public-probability-models which queries
  https://twiki.cern.ch/twiki/bin/view/AtlasPublic.
* Reorder the list by analysis publication date as given by ATLAS.
```

### CODE REVIEW

**Review by matthewfeickert — COMMENTED** [review]
@kratsg if you have time to look at this while I work on PR #1705 that would be helpful. Two things to note:

* There's a mismatch here as we have 26 in this file and https://github.com/pyhf/public-probability-models has 23. :?
* To make things easier to maintain as a human it would be nice to reorder these in reverse chronological order as ATLAS lists them (trying to order them in reverse chronological order by HEPData upload date is too difficult for me to try to figure out where to put changes in).

File: docs/bib/HEPData_likelihoods.bib [hunk 1]
```diff
@@ -151,6 +180,56 @@ @misc{ins1754675
...
+    title = "{Evidence for $t\bar{t}t\bar{t}$ production in the multilepton final state in proton-proton collisions at $\sqrt{s}=13$ $\text {TeV}$ with the ATLAS detector}",
+    doi = "10.17182/hepdata.100170",
+    url = "https://doi.org/10.17182/hepdata.100170",
+    year = "2020",
+    type = "Data Collection",
+    collaboration = "ATLAS",
+}
+
+% 2020-06-24
+@misc{ins1802524,
```
Comment (kratsg) [inline docs/bib/HEPData_likelihoods.bib:214]: why is this added twice?

File: docs/bib/HEPData_likelihoods.bib [hunk 2]
```diff
@@ -151,6 +180,56 @@ @misc{ins1754675
...
+    title = "{Search for Displaced Leptons in $\sqrt{s} = 13$ TeV $pp$ Collisions with the ATLAS Detector}",
+    doi = "10.17182/hepdata.98796",
+    url = "https://doi.org/10.17182/hepdata.98796",
+    year = "2020",
+    type = "Data Collection",
+    collaboration = "ATLAS",
+}
+
+% 2020-10-27
+@misc{ins1827025,
```
Comment (kratsg) [inline docs/bib/HEPData_likelihoods.bib:194]: this is also added twice

File: docs/bib/HEPData_likelihoods.bib [hunk 3]
```diff
@@ -151,6 +180,56 @@ @misc{ins1754675
     collaboration = "ATLAS",
 }
 
+% 2020-11-13
+@misc{ins1831504,
```
Comment (kratsg) [inline docs/bib/HEPData_likelihoods.bib:184]: this is added twice

**Comment by kratsg** [comment]
> * To make things easier to maintain as a human it would be nice to reorder these in reverse chronological order as ATLAS lists them (trying to order them in reverse chronological order by HEPData upload date is too difficult for me to try to figure out where to put changes in).

ATLAS lists as reverse chronological time of publication. I'm not sure what other ordering you're looking for (HEPData is not uploaded when a paper is published, and is typically days/weeks/months after a paper is published).

**Comment by matthewfeickert** [comment]
Okay I think this is fixed now:

```console
$ git grep "%" -- docs/bib/HEPData_likelihoods.bib
docs/bib/HEPData_likelihoods.bib:% 2022-05-05
docs/bib/HEPData_likelihoods.bib:% 2022-04-27
docs/bib/HEPData_likelihoods.bib:% 2021-12-22
docs/bib/HEPData_likelihoods.bib:% 2021-08-17
docs/bib/HEPData_likelihoods.bib:% 2021-06-22
docs/bib/HEPData_likelihoods.bib:% 2021-06-17
docs/bib/HEPData_likelihoods.bib:% 2021-06-03
docs/bib/HEPData_likelihoods.bib:% 2021-03-23
docs/bib/HEPData_likelihoods.bib:% 2021-01-27
docs/bib/HEPData_likelihoods.bib:% 2021-01-05
docs/bib/HEPData_likelihoods.bib:% 2020-11-20
docs/bib/HEPData_likelihoods.bib:% 2020-11-13
docs/bib/HEPData_likelihoods.bib:% 2020-10-27
docs/bib/HEPData_likelihoods.bib:% 2020-07-29
docs/bib/HEPData_likelihoods.bib:% 2020-06-24
docs/bib/HEPData_likelihoods.bib:% 2020-03-26
docs/bib/HEPData_likelihoods.bib:% 2019-12-18
docs/bib/HEPData_likelihoods.bib:% 2019-11-28
docs/bib/HEPData_likelihoods.bib:% 2019-11-15
docs/bib/HEPData_likelihoods.bib:% 2019-09-19
docs/bib/HEPData_likelihoods.bib:% 2019-09-18
docs/bib/HEPData_likelihoods.bib:% 2019-08-21
docs/bib/HEPData_likelihoods.bib:% 2019-08-08
$ git grep "%" -- docs/bib/HEPData_likelihoods.bib | wc -l
23
$ curl -sL https://pyhf.github.io/public-probability-models/atlas.json -o /tmp/atlas.json
$ grep "date" /tmp/atlas.json 
        "date": "2022-05-05",
        "date": "2022-04-27",
        "date": "2021-12-22",
        "date": "2021-08-17",
        "date": "2021-06-22",
        "date": "2021-06-17",
        "date": "2021-06-03",
        "date": "2021-03-23",
        "date": "2021-01-27",
        "date": "2021-01-05",
        "date": "2020-11-20",
        "date": "2020-11-13",
        "date": "2020-10-27",
        "date": "2020-07-29",
        "date": "2020-06-24",
        "date": "2020-03-26",
        "date": "2019-12-18",
        "date": "2019-11-28",
        "date": "2019-11-15",
        "date": "2019-09-19",
        "date": "2019-09-18",
        "date": "2019-08-21",
        "date": "2019-08-08",                                                                                                       
$ grep "date" /tmp/atlas.json | wc -l
23
> diff <(git grep "%" -- docs/bib/HEPData_likelihoods.bib | awk '{print $NF}') <(grep "date" /tmp/atlas.json | awk '{print $NF}' | sed 's/"//g' | sed 's/,//') && echo $?
0
```

**Comment by matthewfeickert** [comment]
Relevant RTD build: https://pyhf--2018.org.readthedocs.build/en/2018/citations.html#published-statistical-models

**Review by kratsg — APPROVED** [review]
